src/kata6.joinNames.js

```diff
@@ -1,3 +1,13 @@
-const joinNames = (namesObj) => {};
 
-module.exports = joinNames;
+const joinNames = (namesObj) => {
+    const array = namesObj.map((person) => person.name);
```

Comment:
Too much indentation, from line 3 til line 10 the whole block has to move back 2 spaces.